internal/slicer/slicer.go

```diff
@@ -36,21 +36,15 @@ func Run(options *RunOptions) error {
 		if path[0] != '/' {
 			panic("bug: tried to add relative path to known paths")
 		}
-		cleanPath := filepath.Clean(path)
-		slashPath := cleanPath
-		if path[len(path)-1] == '/' && cleanPath != "/" {
-			slashPath += "/"
-		}
+		path = fsutil.Clean(path)
 		for {
```

Comment:
I really think this loop can use some comments, taking us through the logic. For example, the ```filepath.Clean()``` is already iterative, so my first question is why do we need the loop.

Reply:
I've added the comment to the (lambda) function:
```
	// addKnownPath path adds path and all its directory parent paths into
	// knownPaths set.
```
This code was introduced the PR from which I linked Gustavo's comment. It's used for checking which paths can be listed by scriptlets.